src/form-layout-grid.scss

```diff
@@ -85,6 +91,13 @@ $cols: 11;
           margin: 0 -0.25rem;
         }
       }
+
+      .#{$fd-namespace}-form-group.#{$blockCol} {
+        padding: 1rem;
+      }
+      .#{$fd-namespace}-form-item.#{$blockRow} {
+        flex-direction: row;
+      }
```

Comment:
Some time ago we decided to not mix blocks in files. Is there any way. Could you create other, stronger element to overwrite `.#{$fd-namespace}-form-group` / `.#{$fd-namespace}-form-item`. Or use them in different elements?

Reply:
![image](https://user-images.githubusercontent.com/26483208/99377241-415a3a80-28c6-11eb-8345-e3a4952de28c.png)

Reply:
so would it be okay to have separate element modifiers such as this?:
![image](https://user-images.githubusercontent.com/53509521/99381487-39fc5680-28f1-11eb-9e0c-f77bce28bec7.png)

Reply:
Yes, it's much better there